core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -42,6 +44,10 @@ oppia.directive('questionsTab', [
             $scope.canEditQuestion = $scope.topicRights.canEditTopic();
             $scope.questionSummaries =
               TopicEditorStateService.getQuestionSummaries();
+            $scope.questionSuggestionThreads = [];
+            $scope.getSuggestedQuestions();
```

Comment:
Um, so, you get the suggested questions, but you don't assign them to anything?

Maybe better to write $scope.loadSuggestedQuestionsAsync() instead.